castor-service/pom.xml

```diff
@@ -17,6 +17,9 @@
         <relativePath>../pom.xml</relativePath>
     </parent>
     <properties>
+        <docker.repository>ghcr.io/carbynestack/${project.artifactId}</docker.repository>
```

Comment:
could you please keep properties grouped and ordered

Reply:
have updated the ordering